<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-TNO9GQV" failures="1" tests="2" name="Test" time="15.676" errors="0" timestamp="2024-08-29T21:23:31 IST">
  <testcase classname="SSquareIT.Day20Framework2.E2ETest2" name="endToEndTest" time="10.332"/>
  <testcase classname="SSquareIT.Day20Framework2.E2ETest2" name="endToEndTest" time="5.289">
    <failure type="java.lang.AssertionError" message="expected [https://tutorialsninja.com/demo/index.php?route=account/account] but found [https://tutorialsninja.com/demo/index.php?route=account/login]">
      <![CDATA[java.lang.AssertionError: expected [https://tutorialsninja.com/demo/index.php?route=account/account] but found [https://tutorialsninja.com/demo/index.php?route=account/login]
at SSquareIT.Day20Framework2.E2ETest2.endToEndTest(E2ETest2.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- endToEndTest -->
</testsuite> <!-- Test -->
